samples/vsphere/appliances/health_messages.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+"""
+* *******************************************************
+* Copyright (c) VMware, Inc. 2019. All Rights Reserved.
+* SPDX-License-Identifier: MIT
+* *******************************************************
+*
+* DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+* EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+* WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+* NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+"""
+
+__author__ = 'VMware, Inc.'
+__vcenter_version__ = '6.7+'
+
+from samples.vsphere.common import (sample_cli, sample_util)
+from com.vmware.appliance_client import Health
+from com.vmware.vapi.std_client import LocalizableMessage
+from com.vmware.appliance_client import Notification
+from vmware.vapi.vsphere.client import create_vsphere_client
+from samples.vsphere.common.ssl_helper import get_unverified_session
+
+
+class HealthMessages(object):
+    """
+    Demonstrates getting Health messages for various health items
+
+    such as memory, storage and cpu
+
+    Retrieves Health messages details from vCenter and prints the data
+
+    """
+
+    def __init__(self):
+        self.item = None
+        self.client = None
+        parser = sample_cli.build_arg_parser()
+
+        parser.add_argument(
+            '--item',
+            required=True,
+            action='store',
+            choices=['memory', 'cpu', 'storage'],
+            help='Specify the name of health item to view the messages')
+
+        args = sample_util.process_cli_args(parser.parse_args())
+        if args.item:
```

Comment:
item argument is Mandatory parameter. I think this check is not required as arg parser it self will throw the error